features/facebookImport/src/locales/en/import.json

```diff
@@ -1,8 +1,8 @@
 {
-    "heading.request": "<strong>Request your data</strong>",
-    "heading.download": "<strong>Download your data</strong>",
-    "heading.import": "<strong>Import your data</strong>",
-    "heading.explore": "<strong>Explore your data</strong>",
+    "heading.request": "Request your data",
```

Comment:
So we're not going to use emphasis in the translated strings anymore?

Reply:
I think this was also due to a wrong font being used here. It's not really according to Figma. Yes, the designers wanted bigger headlines, but the problem was again the wrong font being used. Now with Jost Medium-24 I hope the problem is solved!